src/fabric/src/fabric_doc_update.erl

```diff
@@ -157,7 +163,12 @@ force_reply(Doc, [FirstReply | _] = Replies, {Health, W, Acc}) ->
                         false ->
                             CounterKey = [fabric, doc_update, mismatched_errors],
                             couch_stats:increment_counter(CounterKey),
-                            {error, W, [{Doc, FirstReply} | Acc]}
+                            case check_forbidden_msg(Replies) of
+                                {forbidden, Msg} ->
+                                    {Health, W, [{Doc, {forbidden, Msg}} | Acc]};
+                                false ->
+                                    {error, W, [{Doc, {mismatched_errors, Replies}} | Acc]}
```

Comment:
I wonder if this would break anything since we previously returned a reply but now return the `mismatched_errors` tuple. The `mismatch_errors` is more clear, of course, but it could also break some json serialization of these errors for that doc (if we know how to serialize a reply error shape but not `{mismatched_errors, [_| _] = Replies}` or when we update a single doc with `update_doc(...)`

Reply:
switched back to using FirstReply to make sure things don't